config/karma.conf.js

```diff
@@ -60,8 +60,18 @@ module.exports = function (config) {
       type: 'in-memory'
     },
 
+    sonarQubeUnitReporter: {
```

Comment:
can you move this to process.env.SONAR_QUBE

Reply:
Done